skrf/circuit.py

```diff
@@ -0,0 +1,586 @@
+# -*- coding: utf-8 -*-
+'''
+.. module:: skrf.circuit
+========================================
+circuit (:mod:`skrf.circuit`)
+========================================
+
+
+Provides a class representing a circuit of arbitrary topology,
+consisting of an arbitrary number of N-ports networks or impedance elements.
+
+The results are returned in :class:`~skrf.network.Circuit` objects
+
+
+Circuit Class
+================
+
+.. autosummary::
+   :toctree: generated/
+
+   Circuit
+
+'''
+from . network import Network
+from . media import media
+
+import numpy as np
+import networkx as nx
+import matplotlib.pyplot as plt
```

Comment:
@dvincentwest  moved all plotting to skrf/plotting.py. thinking back i think the idea was allow for non-plotting (headless) installs  and other backends, maybe he can comment on this

Reply:
OK. Should I move the plot function into plotting.py under another name (`plot_graph`) ? 

It's not very clear to me how the method then appear in the Circuit object, I'll look into the code.